cluster/config-defaults.yaml

```diff
@@ -79,8 +79,12 @@ image_policy: "dev"
 {{end}}
 
 # cadvisor settings
-cadvisor_cpu: "150m"
-cadvisor_memory: "150Mi"
+cadvisor_cpu: "125m"
+cadvisor_memory: "125Mi"
+
+# node-problem-detector settings
+node_problem_detector_cpu: "25m"
+node_problem_detector_memory: "25Mi"
```

Comment:
I think `50Mi` is the minimum enforced by our admission controller in other namespaces because it mitigates issues in docker/kubelet. It may be ok to set it lower than `50Mi` @aermakov-zalando WDYT?

Reply:
It should be fine as long as it runs, but we did have problems in the past even with the most basic things (a container running shell+cat) due to how caches are accounted for. Let's see if it works.